README.md

```diff
@@ -545,6 +546,141 @@ module.export = function migrateData(app) {
+
+For example:
+```
+{_id: 'myid', loopback__model__name: 'Foo'}
+{_id: 'myid', loopback__model__name: 'Bar'}
+```
+
+It is recommended to not set ```"forceId": false``` because data belonging to different models can interfere with each other.
+
+Here is an example of a case that would result in a document update conflict using ```"forceId": false```:
+
+### model-definition
+
+```json
+{
+  "name": "Employee",
+  "base": "PersistedModel",
+  "idInjection": true,
+  "forceId": false,
+  "options": {
+    "validateUpsert": true
+  },
+  "properties": {
+    "_id": {
+      "type": "number",
+      "id": true,
+      "required": true
+    },
+    "name": {
+      "type": "string"
+    },
+    "age": {
+      "type": "number"
+    }
+  },
+  "validations": [],
+  "relations": {},
+  "acls": [],
+  "methods": {}
+}
+
+```
+
+```json
+{
+  "name": "Car",
+  "base": "PersistedModel",
+  "idInjection": true,
+  "forceId": false,
+  "options": {
+    "validateUpsert": true
+  },
+  "properties": {
+    "_id": {
+      "type": "number",
+      "id": true,
+      "required": true
+    },
+    "make": {
+      "type": "string"
+    },
+    "model": {
+      "type": "string"
+    }
+  },
+  "validations": [],
+  "relations": {},
+  "acls": [],
+  "methods": {}
+}
+```
+
+### boot-script
+
+```js
+'use strict';
+
+var util = require('util');
+var _ = require('lodash');
+
+module.exports = function(app) {
+  var db = app.datasources.cloudantDs;
+  var Employee = app.models.Employee;
+  var Car = app.models.Car;
+
+  db.once('connected', function() {
+    db.automigrate(function(err) {
+      if (err) throw err;
+      console.log('\nAutomigrate completed');
+
+      Employee.create([{
+        _id: 1,
+        name: 'Foo',
+      }, {
+        _id: 2,
+        name: 'Bar',
+      }], function(err, result) {
+        if (err) throw err;
+        console.log('\nCreated employee instance: ' + util.inspect(result));
+
+        Car.create([{
+          _id: 1,
+          make: 'Toyota',
+        }, {
+          _id: 2,
+          name: 'BMW',
+        }], function(err, result) {
+          if (err) throw err;
+          console.log('\nCreated car instance: ' + util.inspect(result));
+        });
+      });
+    });
+  });
+};
+```
+
+Running the above script will throw a document update conflict because there exists a model instance with `_id` value of 1 and 2.
+
+```bash
+Web server listening at: http://localhost:3000
+Browse your REST API at http://localhost:3000/explorer
+
+Automigrate completed
+Created employee instance: [ { _id: '1', name: 'Foo' }, { _id: '2', name: 'Bar' } ]
+
+/Users/ssh/workspace/sandbox/loopback-sandbox/apps/cloudant/cloudant-forceId-app/server/boot/script.js:33
+          if (err) throw err;
+                   ^
+Error: Document update conflict. (duplicate?),Error: Document update conflict. (duplicate?)
+```
+
+
```

Comment:
To wrap up, you can mention how to correct the above example here. They need to at least set`forceId: true` on one of the model defs or take our recommendation and set it to `true` on both (or dont set it if that is the default behaviour).